src/pages/Patch.tsx

```diff
@@ -34,12 +42,26 @@ export const Patch: React.FC = () => {
   const status = get(patch, "status");
   const description = get(patch, "description");
   const activated = get(patch, "activated");
-
+  useEffect(() => {
+    if (error) {
+      dispatchBanner.error(
+        `There was an error loading the patch: ${error.message}`
+      );
+    }
+  }, [error]);
```

Comment:
You should add dispatchBanner as a dependency for the useEffect hook. 
![image](https://user-images.githubusercontent.com/4605522/81100207-23a12700-8eda-11ea-986e-6c3c21c88081.png)

Reply:
I think if you have a plugin that runs eslint --fix on save it should automatically handle resolving use effect dependencies for you.

Reply:
useQuery also accepts an onError callback in the options where we can dispatch the banner.

Reply:
i left it out on purpose because it calls `dispatchBanner` in an endless loop. i will use the `onError` callback.